nx-X11/programs/Xserver/dix/property.c

```diff
@@ -531,6 +531,7 @@ ProcGetProperty(ClientPtr client)
 	pProp = pProp->next;
     }
 
+    memset(&reply, 0, sizeof(xGetPropertyReply));
```

Comment:
Needs to be backported to `nx-X11/programs/Xserver/hw/nxagent/NXproperty.c`. Also, there are many other places in `NXproperty.c` where events are generated not correctly zero-initialized... maybe do them, too?